packages/config-plugins/src/utils/Updates.ts

```diff
@@ -19,3 +25,82 @@ export function getUpdateUrl(
   }
   return `https://exp.host/@${user}/${config.slug}`;
 }
+
+export function getNativeVersion(
+  config: Pick<ExpoConfig, 'version'> & {
+    android?: Pick<Android, 'versionCode'> & TempRuntimeVersion;
+    ios?: Pick<IOS, 'buildNumber'> & TempRuntimeVersion;
+  },
+  platform: 'android' | 'ios'
+): string {
+  const version = IOSConfig.Version.getVersion(config);
+  switch (platform) {
+    case 'ios': {
+      const buildNumber = IOSConfig.Version.getBuildNumber(config);
+      return `${version}(${buildNumber})`;
+    }
+    case 'android': {
+      const versionCode = AndroidConfig.Version.getVersionCode(config);
+      return `${version}(${versionCode})`;
+    }
+    default: {
+      throw new Error(
+        `"${platform}" is not a supported platform. Choose either "ios" or "android".`
+      );
+    }
+  }
+}
+
+/**
+ * Compute runtime version policies.
+ * @return an expoConfig with only string valued platform specific runtime versions.
+ */
+export const withRuntimeVersion: ConfigPlugin = config => {
+  if (config.ios || config.runtimeVersion) {
```

Comment:
Will we end up with the correct result here if `config.ios` is truthy but both `config.runtimeVersion` and `config.ios.runtimeVersion` are undefined? Wondering if this should be checking `if (config.ios?.runtimeVersion || config.runtimeVersion)` instead.

Reply:
great catch, thank you!